tools/IBM/writeIBMFiles.m

```diff
@@ -853,10 +853,10 @@
 ylim([0 ysize])
 zlim([0 zsize])
 
-%scatter3(X_u(solid_u), Y_u(solid_u), Z_u(solid_u), 10,[0,0,1],'filled')
-%scatter3(X_v(solid_v), Y_v(solid_v), Z_v(solid_v), 10,[0,0,1],'filled')
-%scatter3(X_w(solid_w), Y_w(solid_w), Z_w(solid_w), 10,[0,0,1],'filled')
-%scatter3(X_c(solid_c), Y_c(solid_c), Z_c(solid_c), 10,[0,0,1],'filled')
+scatter3(X_u(solid_u), Y_u(solid_u), Z_u(solid_u), 10,[0,0,1],'filled')
```

Comment:
I would prefer if the preprocessing didn't try to plot anything by default... We could potentially put a switch around this.

Reply:
Commented these lines.